lib/converter.js

```diff
@@ -494,7 +494,7 @@ function detectLibreOffice (additionalPaths) {
   var _pythonName = 'python';
   var _sofficeName = 'soffice';
   var _linuxDirnamePattern = /^libreoffice\d+\.\d+$/;
-  var _windowsDirnamePattern = /^LibreOffice \d+(?:\.\d+)?$/i;
+  var _windowsDirnamePattern = /^LibreOffice( \d+(?:\.\d+)?)?$/i;
```

Comment:
I think this line should be
```
var _windowsDirnamePattern = /^LibreOffice( \d+(?:\.\d+)*?)?$/i;
```
as this would be more permissive - it would be able to catch "LibreOffice", "LibreOffice 6.3", AND "LibreOffice 6.3.4.2" for example.

Reply:
Yes. Thank you